apps/server/src/api-integration/integration.controller.ts

```diff
@@ -156,35 +150,32 @@ const actionHandlers: Record<string, ActionHandler> = {
   },
   /* Extra timers */
   extratimer: (payload) => {
-    if (payload && typeof payload === 'string') {
-      if (payload === SimplePlayback.Start) {
-        const reply = extraTimerService.start();
-        return { payload: reply };
-      }
-      if (payload === SimplePlayback.Pause) {
-        const reply = extraTimerService.pause();
-        return { payload: reply };
-      }
-      if (payload === SimplePlayback.Stop) {
-        const reply = extraTimerService.stop();
-        return { payload: reply };
-      }
-    }
-
     if (payload && typeof payload === 'object') {
-      if ('settime' in payload) {
-        const time = numberOrError(payload.settime);
-        const reply = extraTimerService.setTime(time);
-        return { payload: reply };
-      }
-      if ('direction' in payload) {
-        if (payload.direction === SimpleDirection.CountUp || payload.direction === SimpleDirection.CountDown) {
-          const reply = extraTimerService.setDirection(payload.direction);
-          return { payload: reply };
-        } else {
-          throw new Error('Invalid direction payload');
+      Object.entries(payload).forEach(([key, value]) => {
```

Comment:
I am not really sure I follow the intention here. Why would I be trying to setup multiple timers in the same message? 

I think we should be carefully not to complicate this prematurely